### PR TITLE
Zooming world map with keyboard

### PR DESCRIPTION
**Pull request by Craluminum2413**
Zoom+ with `Page Up` and `+`
Zoom- with `Page Down` and `-`

### CODE REVIEW

File: Systems/WorldMap/GuiElementMap.cs [hunk 1]
```diff
@@ -305,6 +305,22 @@ public override void OnKeyDown(ICoreClientAPI api, KeyEvent args)
         {
             base.OnKeyDown(api, args);
 
+            if (args.KeyCode == (int)GlKeys.PageUp || args.KeyCode == (int)GlKeys.Plus)
```
Comment (Longboyy) [inline Systems/WorldMap/GuiElementMap.cs:308]: Hardcoded keys. Use `ICoreClientAPI.Input.RegisterHotKey`

File: Systems/WorldMap/GuiElementMap.cs [hunk 2]
```diff
@@ -305,6 +305,22 @@ public override void OnKeyDown(ICoreClientAPI api, KeyEvent args)
         {
             base.OnKeyDown(api, args);
 
+            if (args.KeyCode == (int)GlKeys.PageUp || args.KeyCode == (int)GlKeys.Plus)
+            {
+                this.ZoomAdd(
```
Comment (Longboyy) [inline Systems/WorldMap/GuiElementMap.cs:310]: Copy-pasted code can be simplified.

**Review by Longboyy — CHANGES_REQUESTED** [review]
Good change, just needs a few changes to add keybindings and simplification of the copy-pasted code.

**Comment by tyronx** [comment]
this indeed should not be hardcoded

**Comment by Craluminum2413** [comment]
Opened the same PR with fixed problems
https://github.com/anegostudios/vsessentialsmod/pull/10

**Comment by Craluminum2413** [comment]
Oh it is in 1.20 now, I haven't noticed

**Pull request closed without merging**